blackvuesync.py

```diff
@@ -479,7 +484,7 @@ def prepare_destination(destination, grouping):
                 os.remove(outdated_filepath)
 
 
-def sync(address, destination, grouping, download_priority):
+def sync(address, destination, grouping, download_priority, filter_recordings):
```

Comment:
could you change this to `recording_filter` (adj+noun) everywhere? (`filter_recordings` is verb+noun, which fits function names better)